gui/slick/js/new/history.js

```diff
@@ -6,7 +6,7 @@ $(document).ready(function(){
             if(isMeta('sickbeard.HISTORY_LAYOUT', ['detailed'])){
                 return {
                     0: function(node) { return $(node).find('time').attr('datetime'); },
-                    4: function(node) { return $(node).find("span").text().toLowerCase(); }
+                    6: function(node) { return $(node).find("span").text().toLowerCase(); }
                 };
```

Comment:
@alisonatwork  would you please help me with this? Also JS is not rendering compact mode with this PR.